### DISCUSSION TITLE
Welcome to Kochubot Discussions!

### DISCUSSION THREAD

**Discussion by jithumon** [jithumon/tgbot#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vaishnavofficial** [comment]
I would like to contribute a file to tgbot
**A FILE WHICH PRINTS THE TIME **
Can be operated with the command `/time`
